### PR TITLE
feat(suite-native): Simplify biometrics UI logic

### LINKED ISSUE #11511: Change Biometric icons and texts
Change the UI according to [🆕 design](https://www.figma.com/file/RlWH784CRUIvooy8aq6XoR/Main-Flow?type=design&node-id=628-20420&mode=design&t=fJMnB8RW5tcKvkiP-11)  and [this](https://www.figma.com/file/Z6AGVUmKQzLNtDozFamW7f/%5Bs2.0%5D-Mobile?type=design&node-id=22244-2009&mode=design&t=lWEUl9U3b8Jmig3I-0)

- [x] Use the same icon/text for all possible options. 
- [x] Differentiate ios and android according to design.
- [x] Change the order of primary/secondary button so the primary is on top
- [x] Change icon in settings to follow other changes

### PR DESCRIPTION
**Pull request by vytick**
Change the UI according to [🆕 design](https://www.figma.com/file/RlWH784CRUIvooy8aq6XoR/Main-Flow?type=design&node-id=628-20420&mode=design&t=fJMnB8RW5tcKvkiP-11) and [this](https://www.figma.com/file/Z6AGVUmKQzLNtDozFamW7f/%5Bs2.0%5D-Mobile?type=design&node-id=22244-2009&mode=design&t=lWEUl9U3b8Jmig3I-0)

 ✅  Use the same icon/text for all possible options.
 ✅  Differentiate ios and android according to design.
 ✅  Change the order of primary/secondary button so the primary is on top
 ✅  Change icon in settings to follow other changes

Resolve #11511

## Screenshots:

ios:

android:

### CODE REVIEW

File: suite-native/biometrics/src/components/BiometricsIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        padding: ICON_WRAPPER_PADDING,
+        borderRadius: utils.borders.radii.round,
+        backgroundColor: utils.colors.backgroundSurfaceElevation2,
+        color: utils.colors.iconPrimaryDefault,
+        width: wrapperSize,
+        height: wrapperSize,
+
+        extend: {
+            condition: showShadow,
+            style: {
```
Comment (PeKne) [inline suite-native/biometrics/src/components/BiometricsIcon.tsx]: micro nitpick:
```suggestion
            style: utils.boxShadow.small
```

**Review by PeKne — APPROVED** [review]
Looks good 👍

**Comment by PeKne** [comment]
Didn't you upload the iOS demo screenshot twice? The Android and iOS screenshots look identical. But the code changes are quite understandable, so not a big deal :).

File: suite-native/biometrics/src/components/BiometricOverlay.tsx [hunk 2]
```diff
@@ -72,8 +44,8 @@ export const BiometricOverlay = ({
                     onPress={onBiometricAuthPress}
                     style={applyStyle(bottomWrapperStyle)}
                 >
-                    <BiometricsIcons iconSize={32} showShadow />
-                    <Text color="textPrimaryDefault">{<Translation id={titleTransKey} />}</Text>
+                    <BiometricsIcon iconSize={32} showShadow />
```
Comment (juriczech) [inline suite-native/biometrics/src/components/BiometricOverlay.tsx]:
```suggestion
                    <BiometricsIcon iconSize={iconSizes.extraLarge} showShadow />
```

File: suite-native/biometrics/src/components/BiometricsIcon.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+                shadowRadius: utils.boxShadows.small.shadowRadius,
+                elevation: utils.boxShadows.small.elevation,
+                shadowOpacity: utils.boxShadows.small.shadowOpacity,
+            },
+        },
+    }),
+);
+
+type BiometricsIconProps = {
+    iconSize?: number;
```
Comment (juriczech) [inline suite-native/biometrics/src/components/BiometricsIcon.tsx:27]: For now, I think `IconSize` (32) is the only value used to send down to this component as size. We could try to keep it tied to the theme sizes and extend if needed. Not a big deal though..

```suggestion
    iconSize?: IconSize;
```

**Review by juriczech — COMMENTED** [review]
Nice simplification. LGTM

**Comment by vytick** [comment]
https://github.com/trezor/trezor-suite/pull/11910#issuecomment-2041934552

They are slightly different :) The icons look similar, but are not the same

**Comment by PeKne** [comment]
> [#11910 (comment)](https://github.com/trezor/trezor-suite/pull/11910#issuecomment-2041934552)
> 
> They are slightly different :) The icons look similar, but are not the same

Ok, you are right. The Apple dynamic island confused me :D

File: suite-native/biometrics/src/components/BiometricsIcon.tsx (hunk 3 above)
Comment (vytick) [reply suite-native/biometrics/src/components/BiometricsIcon.tsx:27]: Default size is 64, that one is not in icon sizes. And I derive wrapper size from icon size

**Review by juriczech — APPROVED** [review]

File: suite-native/biometrics/src/components/BiometricsIcon.tsx (hunk 1 above)
Comment (vytick) [reply suite-native/biometrics/src/components/BiometricsIcon.tsx]: 0da1bf0587

File: suite-native/biometrics/src/components/BiometricOverlay.tsx (hunk 2 above)
Comment (vytick) [reply suite-native/biometrics/src/components/BiometricOverlay.tsx]: 0da1bf0587

**Comment by vytick** [comment]
@SocketSecurity ignore-all

**Pull request merged**